src/commands/lambda/constants.ts

```diff
@@ -1,46 +1,49 @@
 export const DD_LAMBDA_EXTENSION_LAYER_NAME = 'Datadog-Extension'
 export const EXTENSION_LAYER_KEY = 'extension'
-export const RUNTIME_LAYER_LOOKUP = {
+export const DOTNET_RUNTIME = 'dotnetcore3.1'
+export const LAYER_LOOKUP = {
   [EXTENSION_LAYER_KEY]: DD_LAMBDA_EXTENSION_LAYER_NAME,
+  'dotnetcore3.1': 'dd-trace-dotnet',
   'nodejs12.x': 'Datadog-Node12-x',
   'nodejs14.x': 'Datadog-Node14-x',
   'python3.6': 'Datadog-Python36',
   'python3.7': 'Datadog-Python37',
   'python3.8': 'Datadog-Python38',
   'python3.9': 'Datadog-Python39',
 } as const
-// We exclude the Extension Layer Key in order for the runtime
-// to be used directly in HANDLER_LOCATION.
-export type Runtime = Exclude<keyof typeof RUNTIME_LAYER_LOOKUP, typeof EXTENSION_LAYER_KEY>
-
-export const ARM_RUNTIMES = [EXTENSION_LAYER_KEY, 'python3.8', 'python3.9']
-export const ARM64_ARCHITECTURE = 'arm64'
-export const ARM_LAYER_SUFFIX = '-ARM'
 
 export enum RuntimeType {
+  DOTNET,
+  CUSTOM,
+  JAVA,
   NODE,
   PYTHON,
+  RUBY,
 }
 
-export const RUNTIME_LOOKUP: {[key: string]: RuntimeType} = {
+export const RUNTIME_LOOKUP = {
+  'dotnetcore3.1': RuntimeType.DOTNET,
+  java11: RuntimeType.JAVA,
```

Comment:
Super nit: I would put this in quotes to be consistent with all of the others.

Reply:
It won't let me. Lint is forcing to look like that for some reason.

Reply:
Oh, I bet it's because it does not contain a dot, but the other ones do, which is why you have to quote them. Never mind then!